src/ludamus/static/og-card/index.html

```
@@ -0,0 +1,185 @@
+<!--
+  Source of truth for the 1200x630 link-preview card. scripts/og-image/build.mjs
+  screenshots this page to src/ludamus/static/og-image.jpg; /design/ frames it
+  live, so the panel there cannot drift from what the card is built from.
+
+  Background: Rick Guidice's cylindrical-colony painting for NASA Ames
+  (public domain, NASA), downscaled to 2600px. Outfit is SIL OFL 1.1.
+-->
+<meta charset="utf-8">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)